### ISSUE TITLE
Android 2.2 support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#191]
```
Before posting, please read the following guidelines for posts in the issue
tracker:
http://code.google.com/p/andors-trail/wiki/Forums_vs_issuetracker


What steps will reproduce the problem?
1. Take a snaphshot of the latest svn repository
2. Build in eclipse
3. Run in an android 2.2 configured emulator.

What is the expected output? What do you see instead?
Andors-trail crashes with the following exception:
java.lang.NoSuchMethodError: java.lang.String.isEmpty


What version of the product are you using? On what device?
Andors-trail 0.6.9dev on android 2.2 emulator 

Following patch correct this:

Index: src/com/gpl/rpg/AndorsTrail/resource/ResourceFileParser.java
===================================================================
--- src/com/gpl/rpg/AndorsTrail/resource/ResourceFileParser.java    (röision 77)
+++ src/com/gpl/rpg/AndorsTrail/resource/ResourceFileParser.java    (copie de 
travail)
@@ -255,9 +255,9 @@
        final ArrayList<ActorConditionEffect> addedConditions = new ArrayList<ActorConditionEffect>();
        ResourceObjectArrayTokenizer.tokenize(parts[startIndex + 12], tokenize2Fields, new ActorConditionTypeArrayAppender(actorConditionTypes, addedConditions, false));

-       if (       boostMaxHP.isEmpty() 
-               && boostMaxAP.isEmpty()
-               && moveCostPenalty.isEmpty()
+       if (       boostMaxHP.length() <= 0  
+               && boostMaxAP.length() <= 0 
+               && moveCostPenalty.length() <= 0 
                && combatTraits == null
                && addedConditions.isEmpty()
            ) {

```

Original issue reported on code.google.com by `oli.sa...@gmail.com` on 6 May 2011 at 3:32

**Comment by GoogleCodeExporter** [comment]
```
Great, thanks for the report! This is indeed a bug.
Committed in r78. Thank you for noticing it!
```

Original comment by `oskar.wi...@gmail.com` on 7 May 2011 at 6:13
- Changed state: **Ready**
- Added labels: **Milestone-0.6.9**, **Priority-Critical**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.9 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 29 Jun 2011 at 7:52
- Changed state: **Done**
